app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -39,7 +39,7 @@ def view_toolbar_filename
       'compare_view_tb'
     elsif @lastaction == "drift"
       'drift_view_tb'
-    elsif %w(ems_container).include?(@layout)
+    elsif %w(ems_container).include?(@layout) && %(timeline utilization).exclude?(@display)
```

Comment:
@zeari I have the feeling that we should whitelist rather than blacklist. Probably the toggle buttons are relevant only for the summary (because it's the summary information that can be viewed either as a list of entries or as a dashboard).